python/core/playing-with-time/datetime-object.md

```diff
@@ -89,16 +89,25 @@ print(today.strftime(format))
 ---
 ## Practice
 
-Check if `date_one` is closer to the present than `date_two`:
+Let's say that you have two variables `date_one` and `date_two`:
 
-```python
-import datetime
+```py
+from datetime import datetime
 
+date_one = datetime(2019, 12, 1)
+date_two = datetime(2019, 1, 12)
+```
+
+Complete the gaps so that the following codeblock returns `True`:
+
+```python
 if ??? ??? ???:
```

Comment:
Since `date_one` and `date_two` are the only two variables, might as well fill them in?

Plus the current answers are ambiguous because `date_one > date_two` is the same as `date_two < date_one`

Reply:
hmm yeah, I didn't think about this. making the changes now

Reply:
new PQ:
```
Complete the gaps so that the following codeblock returns `True`:

```python
from datetime import datetime

date_one = datetime(2019, 12, 1)
date_two = datetime(2019, 1, 12)

if ??? ??? ???:
  print(True)
else:
  print(False)

# Output:
# True
```

* `date_one`
* `>`
* `date_two`
* `!=`
* `=`
```

tried to keep it as close to the original as possible. @nem035 what do you think? should I further improve it or create a new question altogether?

Reply:
Sure, that works 👍